admin_guide/configuring_authentication.adoc

```diff
@@ -524,32 +531,12 @@ is configured to trust the `X-Remote-User` header.
   SSLProxyCACertificateFile /etc/pki/CA/certs/ca.crt
   SSLProxyMachineCertificateFile /etc/pki/tls/certs/authproxy.pem
 
-  # Needed for efficient log watching
-  SetEnv proxy-sendchunked 1
-
-  # The value here will be important to tune for your performance needs.  At
-  # some point Apache has to clean up stale connections.  The proxy is unable
-  # to differenciate stale connections with say 'oc log -f' on a pod that
-  # simply isn't outputing anything.  There are likely many other ProxyPass
-  # settings that will need to be adjusted for Production workloads.
-  ProxyTimeout 60
 
   # Insert your backend server name/ip here.
-  ProxyPass / https://ose3-master.example.com:8443/
-  ProxyPassReverse / https://ose3-master.example.com:8443/
-
-  # Requests should be able to access /oauth/token/request and
-  # /oauth/token/display without authentication.  In the case of
-  # /outh/token/display OpenShift will check one of the
-  # ORIGIN_AUTH_REQUEST_HANDLERS to see if the request is authenticated.
-  # Technically it would require authentication for /oauth/token/display simply
-  # by modifying these two ProxyMatch stanzas.
-  <ProxyMatch /oauth/token/.*>
-    Allow from all
-  </ProxyMatch>
+  ProxyPass /oauth/authorize https://openshift:8443/oauth/authorize
+  ProxyPassReverse /oauth/authorize https://openshift:8443/oauth/authorize
```

Comment:
it would also be fantastic if our example could show best practices:
1. a proxy endpoint for challenging clients which requires an X-CSRF-Token header to be present, and challenges with basic auth
2. a proxy endpoint for login clients which is protected by form auth, and provides a URL that clears the session that they could configure logoutURL to point to.